experiment_buddy/experiment_buddy.py

```diff
@@ -207,6 +208,7 @@ def deploy(host: str = "", sweep_yaml: str = "", proc_num: int = 1, wandb_kwargs
 
 
 def _ask_experiment_id(cluster, sweep):
+    logging.info("Name your run in the pop-up window!")
```

Comment:
This line should be moved inside the try block after importing `tkinter.simpledialog`, given that this is the only case where the popup windows is displayed